modern/src/common/selectors.js

```diff
@@ -1,3 +1,7 @@
 export const getIsAdmin = (state) => state.session.user?.administrator;
 
 export const getUserId = (state) => state.session.user?.id;
+
+export const getDevices = (state) => Object.values(state.devices.items);
+
+export const getItemPosition = (itemId) => (state) => state.positions.items[itemId];
```

Comment:
Can we rename with to just `getPosition`? And we don't need to use `itemId` here. It's just position id.

Reply:
Noted